src/style/card.css

```
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
```

```suggestion
  width: 200px;
```

and add box-sizing: border-box rule